app/models/specification_wrapper.rb

```diff
@@ -48,8 +50,30 @@ def validation_errors
         results
       end
 
+      def publicly_accessible?
+        return validate_http if @specification.source[:http]
+        return validate_git if @specification.source[:git]
+        true
+      end
+
+      def validate_http
+        wrap_timeout proc { HTTP.validate_url(@specification.source[:http]) }
+      end
+
+      def validate_git
+        wrap_timeout proc { system('git', 'ls-remote', @specification.source[:git], 'HEAD') }
```

Comment:
Given that it's trunk, it can only be `:tag`, and I'm validating the repo exists - `pod spec lint` will have checked that the tag exists by this point.